### PR TITLE
Add automatic point calculation for match

### PR DESCRIPTION
**Pull request by nikriek**
connects #100

### CODE REVIEW

**Comment by nikriek** [comment]
connects #100

File: spec/models/match_spec.rb [hunk 1]
```diff
@@ -35,4 +35,32 @@
...
+      let(:match) { FactoryBot.build(:match, :empty_points, score_home: 0, score_away: 1) }
+      it { is_expected.to change { match.points_home }.from(nil).to(0) }
+      it { is_expected.to change { match.points_away }.from(nil).to(3)  }
+    end
+
+    context 'team home and team away have an equal score' do
+      let(:match) { FactoryBot.build(:match, :empty_points, score_home: 1, score_away: 1) }
+      it { is_expected.to change { match.points_home }.from(nil).to(1) }
+      it { is_expected.to change { match.points_away }.from(nil).to(1) }
+    end
```
Comment (citron0xa9) [inline spec/models/match_spec.rb:61]: case for "manually entered points not modified" could be added

**Review by citron0xa9 — APPROVED** [review]
Looks good

**Comment by nikriek** [comment]
Yeah true. But it’s not part of the model 😉

**Comment by nikriek** [comment]
@citron0xa9 Who merges first?

File: spec/models/match_spec.rb (hunk 1 above)
Comment (nikriek) [reply spec/models/match_spec.rb:61]: That is not part of the model

File: app/models/match.rb [hunk 2]
```diff
@@ -81,4 +91,19 @@ def adjust_index_by(offset)
...
     save!
   end
+
+  def calculate_points
+    return if !has_scores? || has_points?
+
+    if score_home > score_away
+      self.points_home = 3
+      self.points_away = 0
+    elsif score_home < score_away
```
Comment (manner) [inline app/models/match.rb]: Shouldn't the win, lose and draw points be retrieved from the corresponding event? Or is this only possible after #197 is merged?

**Review by manner — APPROVED** [review]

File: app/models/match.rb [hunk 3]
```diff
@@ -92,18 +92,20 @@ def adjust_index_by(offset)
...
-      self.points_away = 0
+      set_points(3, 0)
     elsif score_home < score_away
-      self.points_home = 0
-      self.points_away = 3
+      set_points(0, 3)
     else
-      self.points_home = 1
-      self.points_away = 1
+      set_points(3, 3)
```
Comment (manner) [inline app/models/match.rb]: should be `set_points(1, 1)` ;)

Comment (nikriek) [reply app/models/match.rb]: Yeah, that was a copy-paste error. Good that we have tests ;)

File: app/models/match.rb (hunk 2 above)
Comment (nikriek) [reply app/models/match.rb]: yes and yes. But I can leave the hardcoded for now and you insert it ;)

**Pull request merged**